doc/sphinx/practical-tools/utilities.rst

```diff
@@ -130,7 +130,11 @@ Note: building with `dune` is experimental.  See :ref:`building_dune`.
 The `_CoqProject` file contains the information needed to generate a makefile
 for building your project.  Your `_CoqProject` file should be in
 the top directory of your project's source tree.  We recommend using the
-:term:`logical name` of the project as the name of the top directory.
+:term:`logical name` of the project as the name of the top directory.  (Make
+sure that `_CoqProject` has no file extension.  On Windows, some tools such as
+Notepad invisibly append `.txt` even when you ask to save the file as `_CoqProject`.
+Also, File Manager doesn't display file extensions.  You may be better off using
+Cygwin tools such as `bash` and `vi`, which always show file extensions.)
```

Comment:
Maybe we should suggest just using (any) command line, rather than a specific one?  cmd.exe also shows extensions

Reply:
Yeah, I guess.